dstep/translator/IncludeHandler.d

```diff
@@ -133,11 +172,33 @@ private:
 
     string isKnownInclude (string include)
     {
-        include = Path.stripExtension(include);
+        include = Path.stripExtension (include);
 
         if (auto r = knownIncludes.find!((k, _) => include.endsWith(k)))
             return r.value;
 
         return null;
     }
-}
\ No newline at end of file
+
+    /// Checks if the given include file name should be converted to an import declaration.
+    bool isConvertableInclude (string include)
+    {
+        // Do not try to convert empty strings, no matter what the pattern says.
+        return include != "" && cast(bool)(matchFirst(include, convertableIncludePattern));
```

Comment:
I don't think it's necessary to explicitly convert to `bool`. Look at the example for `Captures` [1].

[1] http://dlang.org/phobos/std_regex.html#Captures